cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
+
+I don't want to go into all the different types of cloth diaper just yet as
+there are actually a lot of decent resources online already on this, such as
+https://www.thenappygurus.com/types-of-cloth-nappies.html
+
+The whole of the [Nappy Gurus Help & Advice](https://www.thenappygurus.com/help.html)
+section is extremely well put together and very helfpul if you're starting out.
+
+I'm just leaving this diagram here as a reference point to discuss the
+choices we have made (see below).
+
+## Materials and Absorbency
+
+<img width="664" alt="image" src="https://user-images.githubusercontent.com/4185328/169095781-8c3c8f69-3ae5-4e04-ab39-780f198651a5.png">
+
+Cloth diapers hinge on both their speed of and capacity for absorbency.
+
+The only other thing to mention other than the table above is 'Charcoal Bamboo'.
+This is actually not bamboo at all, it's just a fancy name for microfibre and
+equally unhelpful.
+
+### Absorbency capacity
+The more capacity for absorbency, the more liquid a diaper will hold.
+
+Having said this, every baby is different so whilst some can get away with
+very little absorbency, others are considered 'medium to heavy wetters' and
+will need a lot more absorbency, relying much more heavily on hemp inserts
+(typically used in pocket diapers) or boosters (typically layered into other
+types of diapers to increase absorbency).
+
+### Absorbency speed
+This is important for two reasons:
+1. You need to layer your absorbency so that the layers closer to the baby
```

Comment:
Another _looong_ sentence consider splitting:
![image](https://user-images.githubusercontent.com/194400/173175924-398b92ff-bca4-4f91-9ee2-3361baeec7b3.png)

```suggestion
1. Diaper inserts should be layered so that the one in contact with the baby's skin absorbs fastest.
The initial layer wicks away moisture quickly to prevent leaks and discomfort.
The outer layer(s) can have a slower absorbency speed but higher capacity such that liquid is "transferred" from the first to second layer after the urination event. 
```

Reply:
Adding this one to issue #35 as well because the suggestion isn't _technically_ right - you can't put microfiber (the fastest absorbing insert) in contact with the baby's skin as it gives them an almighty rash 👍

Reply:
Inês, you might be projecting words into my suggestion that aren't there.
At no point have I mentioned the word "***microfibre***". 

As for the "almighty rash" ...
Maybe share a link to where what you say was tested empirically. 
If it's anecdotal, then maybe share a photo if there is one.
Because if not, then it's hearsay. i.e. unsubstantiated. 
Basically _most_ of the noise on Facebook posted by unscientific amateurs.
Note: I'm not saying this isn't true. 
Just that I haven't seen any _evidence_. 
The fabric used in microfibre is 
"ultra-fine synthetic fibres such as polyester, polyamide, or polypropylene"
https://sewport.com/fabrics-directory/microfibre-fabric

This is the same stuff they use in _disposable_ diapers.
https://www.sciencedirect.com/topics/engineering/disposable-diaper
![image](https://user-images.githubusercontent.com/194400/174154539-e58a705d-14c3-4293-92b6-118de61fd2eb.png)

These are in contact with sensitive skin on a _massive_ scale.
Perhaps the diaper manufacturers also make nappy rash cream. 💭 
<!--
When I do a product search, I see "Microfiber Inserts for Cloth Diapers Reusable"
with _hundreds_ of 5-star reviews:
https://www.amazon.com/Microfiber-Inserts-Diapers-Reusable-Washable/dp/B00IIFLCFQ
![image](https://user-images.githubusercontent.com/194400/174153209-9030111c-d904-459f-b4f3-bae668fb894f.png)
Obviously I take that with a pinch of salt on Amazon; always read the _negative_ reviews!

> **Note**: I

Side-note: "Inspired by your browsing history" (which I don't have in this incognito browser ...)
**_Adult_ reusable diapers** ... 🙄 
![image](https://user-images.githubusercontent.com/194400/174153754-c6a1bac2-0439-4009-a7aa-306be43eec65.png)
-->